core/src/test/java/org/elasticsearch/index/query/HasChildQueryBuilderTests.java

```diff
@@ -118,11 +116,11 @@ public IndexFieldDataService fieldData() {
     protected HasChildQueryBuilder doCreateTestQueryBuilder() {
         int min = randomIntBetween(0, Integer.MAX_VALUE / 2);
         int max = randomIntBetween(min, Integer.MAX_VALUE);
-        InnerHitsBuilder.InnerHit innerHit = new InnerHitsBuilder.InnerHit().setSize(100).addSort(STRING_FIELD_NAME_2, SortOrder.ASC);
         return new HasChildQueryBuilder(CHILD_TYPE,
                 RandomQueryBuilder.createQuery(random()), max, min,
                 RandomPicks.randomFrom(random(), ScoreMode.values()),
-                randomBoolean()  ? null : new QueryInnerHits("inner_hits_name", innerHit));
+                randomBoolean()  ? null : new InnerHitBuilder().setName("inner_hits_name").setSize(100)
```

Comment:
Can we randomize one or two properties of the InnerHitBuilder, e.g. name or size?